packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -510,7 +513,15 @@ async function generate(requestHost, tenantId) {
             return buffer;
         });
 
-        const filename = `audit-report-${moment().format('yy-MM-DD')}-${v4()}.xlsx`;
+        let filename = '';
+        // for custom periods, add the period name to the filename
+        if (isCustomPeriod) {
+            const reportingPeriod = await getReportingPeriod(periodId, null, tenantId);
```

Comment:
See [above comment](https://github.com/usdigitalresponse/usdr-gost/pull/2282#discussion_r1465786318): can this be set to `undefined` instead of `null` to eliminate the need for an extra check?